finat/tensor_product.py

```diff
@@ -1,5 +1,5 @@
 from functools import reduce
-from itertools import chain, product
+from itertools import chain, product, permutations as itertools_permutations
```

Comment:
Better to just `import itertools` and then use `itertools.permutations` below.

Reply:
Thanks. Fixed!